diff --git a/Schweizer-Messer/sm_boost/include/boost/portable_binary_archive.hpp b/Schweizer-Messer/sm_boost/include/boost/portable_binary_archive.hpp
index 4efa42c..a1c3ce3 100755
--- a/Schweizer-Messer/sm_boost/include/boost/portable_binary_archive.hpp
+++ b/Schweizer-Messer/sm_boost/include/boost/portable_binary_archive.hpp
@@ -28,7 +28,7 @@
 #endif
 
 #include <boost/archive/basic_archive.hpp>
-#include <boost/detail/endian.hpp>
+#include <boost/predef/other/endian.h>
 
 
 namespace boost {
diff --git a/Schweizer-Messer/sm_boost/src/portable_binary_oarchive.cpp b/Schweizer-Messer/sm_boost/src/portable_binary_oarchive.cpp
index afca220..2eedb53 100755
--- a/Schweizer-Messer/sm_boost/src/portable_binary_oarchive.cpp
+++ b/Schweizer-Messer/sm_boost/src/portable_binary_oarchive.cpp
@@ -9,7 +9,7 @@
 //  See http://www.boost.org for updates, documentation, and revision history.
 
 #include <ostream>
-#include <boost/detail/endian.hpp>
+#include <boost/predef/other/endian.h>
 #include <boost/portable_binary_oarchive.hpp>
 
 namespace boost {
